src/main.js

```diff
@@ -0,0 +1,18 @@
+import {NeuralNetworkContext} from './nn/NeuralNetworkContext'
+
+class ML {
+	constructor() {
+    this._nnContext = null;
+  }
+
+  getNeuralNetworkContext() {
```

Comment:
you should use es6 getter

Reply:
This method implements the spec [getNeuralNetworkContext](https://webmachinelearning.github.io/webnn/#dom-ml-getneuralnetworkcontext).